src/tactic/lean_core_docs.lean

```diff
@@ -219,6 +219,9 @@ leaving as new goals the subterms of `A` and `B` which are not definitionally eq
 Example: suppose the goal is `x * f y = g w * f z`. Then `congr` will produce two goals:
 `x = g w` and `y = z`.
 
+This tactic understands subsingletons. If `x y : t`, and an instance `subsingleton t` is
```

Comment:
```suggestion
If `x y : t`, and an instance `subsingleton t` is
```

maybe? I don't think that sentence is necessary.

Reply:
Removed.

My reasoning was that the "If `x y : t` ..." stuff buried the lede, so it was worth restating the subsingleton aspect to draw attention back to it. But perhaps that was an overreaction.